doc/rst/index.rst

```diff
@@ -22,13 +22,15 @@ Chapel Documentation
    Hello World Variants <examples/index>
    Primers <primers/index>
    Language Specification <language/spec/index>
-   builtins/index
    modules/standard
    modules/packages
    modules/layoutdist
    mason-packages/index
    users-guide/index
 
+   builtins/OwnedObject.rst
+   builtins/SharedObject.rst
```

Comment:
This was a temporary fix to prevent links from breaking. These will be removed once the PRs for owned and shared are merged.